### DISCUSSION TITLE
Back to CP search result on save

### DISCUSSION THREAD

**Discussion by hiasl** [craftcms/cms#11246] [Ideas] [answered]
As an agency we have a recurring question from various customers if it is possible to return to a certain search result, after editing an entry. Currently you get back to the basic entries overview.

Steps:
1.) Go to Entries
2.) Submit a search to limit the shown entries
3.) Click on an entry to edit it, then save it
4.) The previous search result including pagination is lost, needs to be resubmitted

Feature Request: Make it possible to return to a search result (maybe even default behavior) after editing an entry.

**Comment by brandonkelly** [comment, marked as answer]
Craft 5.9.0 is out now, which addresses this. Now edit pages always redirect back to the previous page. (#16140)
